examples/wp-rest-api/src/test-wp-rest-api.js

```diff
@@ -32,11 +32,19 @@ export function createApi( fetch = window.fetch ) {
 		}
 
 		selectors = {
-			getPosts: ( getResource, requireResource ) => ( requirement, page = 1, perPage = 10 ) => {
-				const resourceName = `posts-page:{"page":${ page },"perPage":${ perPage }}`;
+			getPostsPage: ( getResource, requireResource ) => ( requirement, params ) => {
+				const paramsString = JSON.stringify( params, Object.keys( params ).sort() );
+				const resourceName = 'posts-page:' + paramsString;
 				const pageIds = requireResource( requirement, resourceName ).data || [];
 				const pagePosts = pageIds.map( id => getResource( `post:${ id }` ).data );
 				return pagePosts;
+			},
+
+			isPostsPageLoading: ( getResource ) => ( params ) => {
+				const paramsString = JSON.stringify( params, Object.keys( params ).sort() );
+				const resourceName = 'posts-page:' + paramsString;
+				const { data, lastRequested, lastReceived = 0 } = getResource( resourceName );
```

Comment:
warning: `0` is a legitimate timestamp. even though in this case _nothing_ should be last received at time `0` but it's technically valid.

can also use `lastReceived = -Infinity` to be more correct without resorting to `null` values and the like

Reply:
Thanks for the tip!